lib/active_merchant/billing/gateways/bogus.rb

```diff
@@ -118,6 +111,54 @@ def unstore(reference, options = {})
 
       private
 
+      def authorize_emv(money, paysource, options = {})
+        money = amount(money)
+        case money
+        when /00$/
+          Response.new(true, SUCCESS_MESSAGE, {:authorized_amount => money}, :test => true, :authorization => AUTHORIZATION, :emv_authorization => "8A023030" )
```

Comment:
`"8A023030"` should be a constant so it reads more easily to those not familiar with EMV.